data/templates/centos-ks

```diff
@@ -104,6 +104,8 @@ echo -e "NETWORKING=yes\nHOSTNAME=<%=hostname%>.<%=domain%>" > /etc/sysconfig/ne
       <% if( undefined != n.ipv4.vlanId ) { %>
         <% n.ipv4.vlanId.forEach(function(vid) { %>
           echo "Configuring vlan <%=vid%> on <%=n.device%>"
```

Comment:
These sed commands aren't operating on the correct file. They should be operating on <%=n.device%>.<%=vid%>